//test error

int main(){

	int func();
	int x = 5;
	return x;
}
